python_ta/checkers/missing_space_in_doctest_checker.py

```diff
@@ -0,0 +1,61 @@
+"""Checker for E9973 missing-space-in-doctest"""
+import astroid
+
+from pylint.interfaces import IAstroidChecker
+from pylint.checkers import BaseChecker
+from pylint.checkers.utils import check_messages
+
+DOCTEST = ">>>"
+
+
+class MissingSpaceInDoctestChecker(BaseChecker):
+    __implements__ = IAstroidChecker
+
+    name = 'missing_space_in_doctest'
+    msgs = {'E9973': ('Space missing in doctest(s) in the docstring of function "%s."',
```

Comment:
We can make this more specific by saying "Space missing after >>>". Also, use singular "doctest" only, I think in general the message style is to refer to a single occurrence of the error.